### PR TITLE
react rules quickstart

### PR DESCRIPTION
**Pull request by alemata**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

**Comment by alemata** [comment]
@flaviogranero could you please?

File: articles/client-platforms/react/06-rules.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+<%= include('../../_includes/_github', {
+  link: 'https://github.com/auth0-samples/auth0-react-sample/tree/master/06-Rules',
+}) %>_
+
+<%= include('../_includes/_rules-introduction') %>
+
+
+### Test rule result
```
Comment (flaviogranero) [inline articles/client-platforms/react/06-rules.md]: @alemata do you think we can follow the titles format from other tutorials? I'm always numerating and using a h2, something like `## 1. title`

**Comment by alemata** [comment]
@flaviogranero done!

**Comment by flaviogranero** [comment]
👍 @ntotten

**Comment by mpaktiti** [comment]
Please remove the pending label once the PR is ready to be merged.

**Comment by alemata** [comment]
👍 @mpaktiti

File: articles/client-platforms/_includes/_rules-introduction.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Rules are one of the cool features of Auth0. The reason behind that coolness is their flexibility, which gives you the ability to extend what Auth0 has to offer. They are just JavaScript functions which work like middlewares. To see a detailed description, please refer to [the full documentation](https://auth0.com/docs/rules).
 
-### Create a rule
+${title}
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_rules-introduction.md]: Does not display correctly:

image

https://auth0-docs-content-pr-1763.herokuapp.com/docs/quickstart/spa/angular2/06-rules

Comment (mpaktiti) [inline articles/client-platforms/_includes/_rules-introduction.md]: Both images are broken:

image

File: articles/client-platforms/angular2/06-rules.md [hunk 3]
```diff
@@ -33,4 +32,4 @@ Then, display user profile's new `country` attribute added in the created rule a
 ![Country rule sample](/media/articles/angularjs2/rule-country-show.png)
```
Comment (mpaktiti) [inline articles/client-platforms/angular2/06-rules.md:36]: Image is broken https://auth0-docs-content-pr-1763.herokuapp.com/docs/quickstart/spa/angular2/06-rules#test-rule-result

File: articles/client-platforms/react/06-rules.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+---
```
Comment (mpaktiti) [inline articles/client-platforms/react/06-rules.md:1]: Cannot see this page in the deployed app: https://auth0-docs-content-pr-1763.herokuapp.com/docs/quickstart/spa/react

You probably need to update the `index.yml` with the new page: https://github.com/auth0/docs/blob/master/articles/client-platforms/react/index.yml

**Comment by mpaktiti** [comment]
@alemata Hey, I reported some issues with the PR. Please look into them and let me know when fixed.

File: articles/client-platforms/angular2/06-rules.md (hunk 3 above)
Comment (alemata) [inline articles/client-platforms/angular2/06-rules.md:36]: the file is working in prod: https://auth0.com/docs/quickstart/spa/angular2/06-rules
I think images are not working in your test env, cause I don't see any image there..

**Comment by alemata** [comment]
@mpaktiti updated! Could you check please?

**Comment by mpaktiti** [comment]
@alemata Please resize all images to be maximum 750 pixels wide.

**Comment by alemata** [comment]
@mpaktiti done!

File: articles/client-platforms/_includes/_rules-create-section.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+To create a rule just go to [rule new page](https://manage.auth0.com/#/rules/new). You can create it from scratch or use an existing template. These templates are written by Auth0 team to assist you complete common tasks. 
+Let's use the one called *'Add country to the user profile'*
+
+![Empty rule](/media/articles/rules/rule-choose-add-country-template.png)
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_rules-create-section.md]: @alemata Could you please resize these 2 images as well? They are both 1212 × 853 pixels. Thank you!

- https://github.com/auth0/docs/tree/master/media/articles/rules/rule-choose-add-country-template.png
- https://github.com/auth0/docs/tree/master/media/articles/rules/rule-create-add-country-country.png

Comment (alemata) [inline articles/client-platforms/_includes/_rules-create-section.md:4]: @mpaktiti already done in the [commit](https://github.com/auth0/docs/pull/1763/commits/79b3be7c00ed30615f1c0f4f7dc6036844c5fe0c)

Comment (alemata) [inline articles/client-platforms/_includes/_rules-create-section.md:4]: Is that ok? thanks!

File: articles/client-platforms/_includes/_rules-introduction.md [hunk 6]
```diff
@@ -1,18 +1 @@
-Rules are one of the cool features of Auth0. The reason behind that coolness is their flexibility, which gives you the ability to extend what Auth0 has to offer. They are just JavaScript functions which work like middlewares. To see a detailed description, please refer to [the full documentation](https://auth0.com/docs/rules).
```
Comment (mpaktiti) [inline articles/client-platforms/_includes/_rules-introduction.md:1]: @alemata Please use relative URL here: `/rules`

**Pull request merged**